doc/releases_and_migration/migration_guide_addon_v010.rst

```diff
@@ -15,78 +15,39 @@ The Sidewalk release cycle is now separate from the nRF Connect SDK, allowing fo
 You can now find Amazon Sidewalk under the `nRF Connect SDK Add-ons`_ index.
 This solution enables an alternative setup method using the `nRF Connect for VS Code`_.
 
-Prerequisites
-*************
-
-Before you begin the migration process, you must have the following tools installed:
-
-* Git
-* Python 3.8 or later
-* The `west`_ tool
-* `nRF Util`_
-
 Migration steps
 ***************
 
 To successfully migrate to Amazon Sidewalk Add-on you must choose the migration method.
 If you wish to set up a clean repository, follow one of the methods outlined on the :ref:`setting_up_sdk_sidewalk_repository` page.
 If you already have an nRF Connect SDK setup with Sidewalk and wish to migrate, you must complete the following steps:
 
-#. Remove the :file:`.west` directory to clean the west configuration.
+1. Disable Sidewalk group in the manifest
 
    .. code-block:: console
 
-      rm -rf .west
+      west config manifest.group-filter -- "-sidewalk"
 
-#. Checkout and pull main branch in the Sidewalk repository.
+#. Use Sidewalk as a workspace application repository
 
    .. code-block:: console
 
-      cd sidewalk
-      git checkout main
-      git pull origin main
-      cd ..
-
-#. Initialize west with local manifest.
+      west config manifest.path sidewalk
 
-   .. code-block:: console
-
-      west init -l sidewalk
 
-#. Update all repositories, by running the following command:
+#. Update Repositories
 
    .. code-block:: console
 
       west update
 
-   Depending on your connection, the update might take some time.
-
-#. Execute the following commands to get the toolchain hash and download the necessary toolchain:
-
-   .. code-block:: console
-
-      nrf/scripts/toolchain.py
-      nrfutil toolchain install --ncs-version $(nrf/scripts/toolchain.py --ncs-version)
-
-#. Install the required Python packages for both nRF Connect SDK and Zephyr by running the following commands:
-
-   .. code-block:: console
-
-      pip install -r nrf/scripts/requirements.txt
-      pip install -r zephyr/scripts/requirements.txt
```

Comment:
@annwoj I think we repeat nRF Connect documentation here,
Same in doc/setting_up_sidewalk_environment/setting_up_sdk.rst
But please let me know if this step is important and I shouldn't remove it

Reply:
But is that just removing or repeating? Is this performed at some point when following the https://docs.nordicsemi.com/bundle/ncs-2.9.0/page/nrf/installation/install_ncs.html instructions? If so, then that would be unnecessary, but I can't find this part

Reply:
We reference in https://github.com/nrfconnect/sdk-sidewalk/blob/main/doc/setting_up_sidewalk_environment/setting_up_sdk.rst
I assumed  the person reading "migration guide" has already set up, and at least build sidewalk application before.